### ISSUE TITLE
icc profiles

### ISSUE THREAD

**Issue by JanHouserek** [4lex4/scantailor-advanced#15]
Would it be possible to implement at least an option to keep icc profiles embedded in images? ScanTailor currently removes icc profiles by default with no setting regarding icc profiles, if I am not mistaken. Thanks.

**Comment by ismay7** [comment]
Good point. We, too (using ST professionally at the library), would be greatly interested in having this option. Our workaround currently is to convert to sRGB beforehand and re-append it afterwards. So an integrated automatic conversion (to sRGB) would already be pleasant, too. Best regards to everyone here ;-)

**Comment by JanHouserek** [comment]
Thanks for the response. Yes, it is unproblematic to re-attach desired icc profile using ImageMagick. Some feature to keep input icc profiles in output images (and/or more) would come in handy though, as it would safe some invaluable time in our workflow. Seems to make sense in the case you just want to crop & deskew (possibly rotate & split) without any other changes to your original images. How difficult it is to implement, I do not know, it is completely up to you of course. I appreciate your great work on ST anyway, thank you! :)

**Comment by ismay7** [comment]
I guess the only slightly problematic issue here is the ST output not having a color profile attached at all, which is totally ok, as long as one is aware of that fact and knows how to re-attach the one it had at the input stage. If the input images have some certain color profile embedded, for instance a specific scanning device profile, the color accuracy within the ST output images may not be rendered correctly without re-attaching the old profile. It's not a big issue however as long as one is aware of the fact. But STA is coming along superbly, anyway. Thank you for that!

**Comment by 4lex4** [comment]
If you mean 16 bit color depth, to support that almost all the processing algorithms at the output stage modifying the image have to be reworked. It promises to take very much time and I think it isn't worth that.

**Comment by JanHouserek** [comment]
Our standard is 8 bit.
